jung-algorithms/src/main/java/edu/uci/ics/jung/layout/model/DefaultLayoutModelChangeSupport.java

```diff
@@ -6,6 +6,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * an Event system for LayoutModels to announce the change in the location of Nodes This replaces
```

Comment:
missing punctuation

Reply:
I'm we-working the event structure for a future PR